plugins/feeds/public/abusech_malwarebazaar.py

```diff
@@ -1,42 +1,65 @@
 import logging
 from datetime import timedelta, datetime
 
-from core.errors import ObservableValidationError
-from core.feed import Feed
-from core.observables import Hash, File
+from core.schemas import observable
+from core.schemas import task
+from core import taskmanager
+import pandas as pd
+from io import StringIO
 
 
-class AbuseCHMalwareBazaaar(Feed):
+class AbuseCHMalwareBazaaar(task.FeedTask):
     default_values = {
         "frequency": timedelta(hours=1),
         "name": "AbuseCHMalwareBazaaar",
-        "source": "https://bazaar.abuse.ch/export/csv/recent/",
         "description": "This feed contains md5/sha1/sha256",
     }
 
-    def update(self):
+    URL_FEED = "https://bazaar.abuse.ch/export/csv/recent/"
+
+    def run(self):
         data = {"query": "get_recent", "selector": "time"}
-        for _, block in self.update_csv(
-            filter_row="first_seen_utc",
+
+        response = self._make_request(self.URL_FEED, auth=None, verify=True)
+
+        if response.status_code != 200:
+            return False
```

Comment:
Do we want to raise an exception here (maybe with `status_code` and `response.text`), so the error gets reported to the UI?